lib/screens/creategame.dart

```diff
@@ -96,7 +100,10 @@ class _CreateGame extends State<CreateGame> {
                             onTap: () {
                               Navigator.push(
                                 context,
-                                MaterialPageRoute(builder: (context) => CreateRoom(audioController)),
+                                MaterialPageRoute(
+                                  builder: (context) =>
+                                  CreateRoom(audioController, playerId)
+                                ),
```

Comment:
How are we getting the player id before even creating a room. That would mean being connected to the socket everytime which is not needed. Socket connection should be initialised during create or join game

Reply:
Fixed